website/templates/project/project_base.mako

```diff
@@ -14,6 +14,11 @@
 
 <%include file="project_header.mako"/>
 
+% if status and node:
+    ## WARNING: Need new heading alert messages? They must be added at both here and base.mako
+    <%include file="../alert.mako"/>
```

Comment:
@sloria Basically, it is controlled by the if statement. When the page has node(project), the alert message should be shown before project_header.mako. Otherwise, alert message is created on base.mako.